src/layout/components/Aside/Mobsidebar.tsx

```diff
@@ -0,0 +1,45 @@
+import MenuItem from './components/Mitem';
+import ButtomImg from './components/ButImg';
+import '../../../assets/styles/App.css';
+import logo from '../../../assets/images/logo.png';
+import { IoIosClose } from 'react-icons/io';
+
+const Mobsidebar = () => {
+  const items = [
+    { icon: '➡️', label: 'Dashboard' },
+    { icon: '☰', label: 'Transactions' },
+    { icon: '⬆️', label: 'Cards' },
+    { icon: '☰', label: 'Reports' },
+    { icon: '…', label: 'Calendar' },
+    { icon: '☰', label: 'Settings' },
+    { icon: '…', label: 'Chat' }
+  ];
+  function closeNav() {
```

Comment:
use arrow function instead of normal function, `const closeNav = () => {...}`, and same for all future functions